packages/discovery-provider/ddl/migrations/0140_fix_handle.sql

```diff
@@ -0,0 +1,27 @@
+begin;
+
+update users
+set handle = 'cf935d736d'
+where handle_lc = 'hopelessrecords';
```

Comment:
why are all the updates separated?

Reply:
was just trying to make it sing-along-like!

actually just thought it was more readable this way. it's fast enough 🤷